pkg/chain/chain.go

```diff
@@ -88,13 +88,20 @@ type BondedECDSAKeep interface {
 		signature *ecdsa.Signature,
 	) error // TODO: Add promise *async.SignatureSubmissionPromise
 
-	// OnKeepClosed is a callback that is invoked when a notification is sent
-	// about the closed keep.
+	// OnKeepClosed installs a callback that will be called on closing the
```

Comment:
Should we also update docs for all other `On*` functions for consistency to replace `is` with `installs`?

Reply:
Please see https://github.com/keep-network/keep-ecdsa/pull/333/commits/26fb21ef3cf006eb37e786737ba2cf92d117d53a